addons/website_slides/models/slide_channel.py

```diff
@@ -101,9 +101,11 @@ def _send_completed_mail(self):
             if template:
                 records = template_to_records.setdefault(template, self.env['slide.channel.partner'])
                 records += record
+                template_to_records.update({template: records})
 
+        record_email_values = dict()
         for template, records in template_to_records.items():
-            record_email_values = template.generate_email(self.ids, ['subject', 'body_html', 'email_from', 'partner_to'])
+            record_email_values = template.generate_email(records.ids, ['subject', 'body_html', 'email_from', 'partner_to'])
```

Comment:
You have to update the dict; here you keep only the values from the last iteration ^^ .

Reply:
Damn yes, this is fixed now.
Thanks @tde-banana-odoo :sweat_smile: